packages/loader/container-loader/src/container.ts

```diff
@@ -1201,12 +1201,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this._existing = details.existing;
         }
 
-        // LoadContext directly requires protocolHandler to be ready, and eventually calls
-        // instantiateRuntime which will want to know existing state.
         this._protocolHandler = await protocolHandlerP;
 
         const codeDetails = this.getCodeDetailsFromQuorum();
-        await this.loadContext(codeDetails, attributes, snapshot, pendingLocalState);
+        await this.instantiateContext(
+            codeDetails,
+            attributes,
+            this._existing === true,
```

Comment:
Would be great to deprecate existing() property as well (in this or future PRs)

Reply:
Yes, need to remove it's usage from a bunch of places outside of the loader though..

Reply:
Sure. BTW, outside usage might have different semantics (i.e. assumptions that are not actually matched to reality)